data/faqs.toml

```diff
@@ -12,3 +12,7 @@ description = "Can't find the answer you're looking for? We've shared some of ou
 [[faq]]
     question = "What is the branding guide for PyMCon?"
     answer = "The branding guide can be found 👉 <a href='https://github.com/pymc-devs/brand' target='_blank'> here </a>"
+
+[[faq]]
```

Comment:
not very convinced on the actual question and answer but I do want to have one faq about the code of conduct encouraging people to report and against the "code of conduct violations are super grave/lifetime expulsion grave" pre-conception.

Reply:
Thats fine adding the section in as a start is great